stylelint.config.cjs

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  extends: [
+    '@extensionengine/stylelint-config'
+  ],
+  overrides: [
+    {
+      files: ['src/**', 'example/**'],
+      rules: {
+        'selector-pseudo-class-no-unknown': null,
+        'selector-class-pattern': null,
+        'value-keyword-case': null
```

Comment:
What is triggering this rule, can we fix issues instead of disabling the rule?

Reply:
Removed.